packages/components/src/Tabs/Tab.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useRef } from 'react';
+import { Box, useStateProps } from '@marigold/system';
+import { useTab } from '@react-aria/tabs';
+import { useFocus, useHover } from '@react-aria/interactions';
+import { mergeProps } from '@react-aria/utils';
+import { useTabContext } from './Context';
+import { Node } from '@react-types/shared';
+import { TabListState } from '@react-stately/tabs';
+
+export interface TabProps {
+  item: Node<object>;
+  state: TabListState<object>;
+}
+
+export const Tab = ({ item, state }: TabProps) => {
+  const { key, rendered } = item;
+  const ref = useRef(null);
+  const { tabProps, isSelected } = useTab({ key }, state, ref);
+  const disabled = tabProps['aria-disabled'];
```

Comment:
Rather use the tab context du pass down the disabled props from the `Tabs` :)

Reply:
Ok i'll try this

Reply:
ping :)